appveyor.yml

```diff
@@ -15,9 +15,9 @@ cache:
 
 install:
 - ps: >-
-    (new-object net.webclient).DownloadFile('https://download.microsoft.com/download/A/2/D/A2D8587D-0027-4217-9DAD-38AFDB0A177E/msxml.msi', 'C:\\msxml.msi')
+    (new-object net.webclient).DownloadFile('https://download.microsoft.com/download/a/4/5/a45cb3d5-2268-4d13-9efb-ec800262ac98/msxml6.msi', 'C:\\msxml6.msi')
```

Comment:
There is a lot of noise in this PR. There are 5 commits from previous fixes. Any chances you can rebase?
Also CI is broken because you changed the name of the file being downloaded but not of the file being installed